lib/pwa.js

```diff
@@ -122,11 +122,12 @@ exports.findByManifestUrl = function(manifestUrl) {
  *
  * Steps:
```

Comment:
This is super horrible. We should separate out the parts. (The fetching and the validation, at least.)